src/routes/governance/proposals-list.tsx

```diff
@@ -44,11 +44,14 @@ export const ProposalsList = ({ proposals }: ProposalsListProps) => {
             </p>
           </header>
         </Link>
-        <table className="proposal-table">
+        <table
+          className="proposal-table"
+          data-test-id="governance-proposal-table"
```

Comment:
can all `data-test-id`s be changed to `data-testid` the reason for this is Jest uses the latter so we can use the ids in both unit and e2e tests if they are in the latter form but not in the former.

Reply:
Done